oemof_b3/tools/geo.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Point
+
+
+def load_regions_file(file_path):
+    """
+    loads a geopackage containing all regions of Germany
+
+    Input
+    ---------------
+    'file_path': string
+        with path to geopackage containing geoinformation for all regions in Germany
+
+    Outputs
+    --------------
+    geopandas.GeoDataFrame
+        with geoinformation of all regions in Germany
+    """
```

Comment:
```suggestion
    r"""
    Loads a geopackage containing all regions of Germany.

    Parameters
    ---------------
    'file_path' : string
        Path to geopackage containing geoinformation for all regions in Germany

    Returns
    ----------
    regions_file : geopandas.GeoDataFrame
        Geoinformation of all regions in Germany
    """
```

Reply:
Some correction to meet numpy-style convention. See e.g. here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html
Applies to all functions here.